changelog/README.md

```diff
@@ -4,6 +4,7 @@
 
 ### 4.4.0 (in progress)
 
+- [improvement] JAVA-2530: Expose shortcuts for version 3 UUIDs
```

Comment:
why do we need to support both 3 and 4 versions of UUIDs in the same driver version?

Reply:
We don't need to, we are just trying to be nice to users and provide them with a user-friendly API to create most kinds of UUIDs. We already had methods to create time-based UUIDs (version 1) and random UUIDs (version 4), now we are also offering methods to create name-based UUIDs (versions 3 and 5).